### PR TITLE
Fixes 117: Container deployment files

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMSCONTENT-117

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/14733751

**Comment by Andrewgdewar** [comment]
I'm not confident enough to review this ticket, would @avisiedo be a good person to review this?

**Comment by jlsherrill** [comment]
probably @mhriver, although once we are onboarded, he may just take it over and get the pipelines setup for building.  Feel free to just let this be for now :)

File: build_deploy.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# --------------------------------------------
+# Export vars for helper scripts to use
+# --------------------------------------------
+# name of app-sre "application" folder this component lives in; needs to match for quay
+export COMPONENT="content-sources"
```
Comment (mshriver) [inline build_deploy.sh:7]: I think its supposed to be:

```
APP_NAME="content-sources"  # name of app-sre "application" folder this component lives in
COMPONENT_NAME="content-sources-frontend"  # name of app-sre "resourceTemplate" in deploy.yaml for this 
```

Comment (mshriver) [reply build_deploy.sh:7]: I'm going to disagree with myself, after seeing Ondrej's work on provisioning-frontend.

**Review by mshriver — APPROVED** [review]

**Comment by mshriver** [comment]
What's not yet part of CI can't hurt us :)

**Pull request merged**